packages/eslint-plugin/src/rules/prefer-includes.ts

```diff
@@ -0,0 +1,214 @@
+/**
```

Comment:
Sorry we've changed to using all-contributors now and removed all other occurrences of these file-level author comments, please could you remove this